pages/posts/[id].jsx

```diff
@@ -0,0 +1,51 @@
+import axios from "axios";
+import Link from "next/link";
+import { Button } from "../../components/Button";
+import Post from "../../components/Post";
+import { StyledSection } from "../../components/Posts/Posts.styled";
+
+const Detais = ({ post }) => {
+  return (
+    <>
+      <Button as={Link} href="/" padding="9px 15px" radius="5px">
+        Zurück
+      </Button>
+      <StyledSection>
+        <Post
+          image={post?.image}
+          title={post?.title}
+          alt={post?.alt}
+          full_desc={post?.full_desc}
+        />
```

Comment:
It might be a better idea to only render the `Post` component conditionally if `post` is defined:

```suggestion
    {post && (
        <Post
          image={post.image}
          title={post.title}
          alt={post.alt}
          full_desc={post.full_desc}
        />
        )}
```

Reply:
yes yYou're right